index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8"/>
+        <link rel="stylesheet"
+            href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css"
+            integrity="sha512-+4zCK9k+qNFUR5X+cKL9EIR+ZOhtIloNl9GIKS57V1MyNsYpYcUrUeQc9vNfzsWfV28IaLL3i96P9sdNyeRssA=="
+            crossorigin="anonymous" />
+        <link rel="stylesheet" href="./styles.css">
+    </head>
+    <body>
+        <!--<h1>Hello, we are Joel and Geofrey</h1>-->
```

Comment:
I think it would be better to leave unused code in your code 😄.

unused code can cause error on future and semantic readability 👍.

Reply:
- [ ] Thank you for showing your good intention to help the student :wink: but in a bid to communicate this good gesture to the student, it is imperative to make sure that there is no potential interference or distortion that can confuse the student. Unclear communication leaves a student blocked rather than getting helped. Can you please look at your statement again? I think you wanted to write: "I think it would be better `NOT` to leave unused code in your code", instead you said it better to... and that is what the student did. Kindly check this and restructure it.
- [ ] Also, you did a good job explaining why the change was necessary but again, it really isn't clear due to some choice of grammar. I think it would be more helpful to read your statements out loud until you are convinced you understood what the message is about :+1:  
_I think you meant to suggest to the student that: `Unused code cause error (introduce a bug) in the future and affect the readability of the code`_. Kindly notice how my first letter "U" was capitalized.
- Clear conversation and unambiguous communication is very important in code review
Good job with the emojis :hugs: